themes/bootstrap3/less/components/search.less

```diff
@@ -438,6 +438,10 @@ body.rtl {
 .combined-search-container.grid {
   margin-left: 1rem;
   margin-right: 1rem;
+
+  .combined-list {
+    overflow-x: auto;
+  }
```

Comment:
I don't love the horizontal scrollbar, but it's better than overlapping content.  clip or hidden would be option if @crhallberg prefers.

Reply:
I also agree that the overflow scrollbar is not good. I'd prefer hidden but there's more we can do, I think. Since this happens when long words come into play, we can allow for more breaks in words:

```css
.combined-list {
    word-break: break-word;
    hyphens: auto;
}
```

We can also do [what we do locally at Villanova](https://library.villanova.edu/Find/Combined/Results?type=&lookfor=behold&limit=20&sort=relevance) and make the container on the combined screen full width.